src/SUMMARY.md

```diff
@@ -14,3 +14,5 @@
 - [Morello known issues](issues/README.md)
 - [Getting help](support/README.md)
 - [Resources](resources/README.md)
+
+[Current release: 22.05]()
```

Comment:
I'd prefer if this text was better visible and used a variable from `book.toml` but I didn't find a way to do that.